### ISSUE TITLE
About run rl_env_example.py

### ISSUE THREAD

**Issue by Tfly6** [ntnu-arl/aerial_gym_simulator#55]
When I run the `python3 rl_env_example.py`, I see the following errors:
```bash
[3496 ms][robot_manager] - CRITICAL : If your robot differs across environments you need to perform this computation for each different robot here. (robot_manager.py:433)
...
[4761 ms][robot_manager] - ERROR : Not using warp. Initializing sensors `(robot_manager.py:131)
```
But the program did not exit abnormally. Does the errors have any impact?

<details>
<summary>Full Terminal output</summary>
Importing module 'gym_38' (/home/tfly/src/isaacgym/python/isaacgym/_bindings/linux-x86_64/gym_38.so)
  Setting GYM_USD_PLUG_INFO_PATH to /home/tfly/src/isaacgym/python/isaacgym/_bindings/linux-x86_64/usd/plugInfo.json
  PyTorch version 2.3.0.post301
  Device count 1
  /home/tfly/src/isaacgym/python/isaacgym/_bindings/src/gymtorch
  Using /home/tfly/.cache/torch_extensions/py38_cu118 as PyTorch extensions root...
  Emitting ninja build file /home/tfly/.cache/torch_extensions/py38_cu118/gymtorch/build.ninja...
  Building extension module gymtorch...
  Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
  ninja: no work to do.
  Loading extension module gymtorch...
  Warp 1.0.0 initialized:
     CUDA Toolkit 11.5, Driver 12.2
     Devices:
       "cpu"      : "x86_64"
       "cuda:0"   : "NVIDIA GeForce RTX 3060" (12 GiB, sm_86, mempool enabled)
     Kernel cache:
       /home/tfly/.cache/warp/1.0.0
  Gym has been unmaintained since 2022 and does not support NumPy 2.0 amongst other critical functionality.
  Please upgrade to Gymnasium, the maintained drop-in replacement of Gym, or contact the authors of your software and request that they upgrade.
  See the migration guide at https://gymnasium.farama.org/introduction/migration_guide/ for additional information.
  [2076 ms][__main__] - INFO : An Info message will look like this (logging.py:50)
  [2076 ms][__main__] - WARNING : A Warning message will look like this (logging.py:51)
  [2076 ms][__main__] - ERROR : An Error message will look like this (logging.py:52)
  [2076 ms][__main__] - CRITICAL : A Critical message will look like this (logging.py:53)
  [2076 ms][__main__] - WARNING : 
  
  
  Just an example task interface.
  
  
   (rl_env_example.py:10)
  [2076 ms][base_task] - INFO : Setting seed: 1 (base_task.py:38)
  [2077 ms][position_setpoint_task] - INFO : Building environment for position setpoint task. (position_setpoint_task.py:43)
  [2077 ms][position_setpoint_task] - INFO : 
  Sim Name: base_sim,
  Env Name: empty_env,
  Robot Name: base_quadrotor, 
  Controller Name: lee_attitude_control (position_setpoint_task.py:44)
  [2077 ms][position_setpoint_task] - INFO : 
  Num Envs: 4096,
  Use Warp: False,
  Headless: False (position_setpoint_task.py:52)
  [2077 ms][env_manager] - INFO : Populating environments. (env_manager.py:73)
  [2077 ms][env_manager] - INFO : Creating simulation instance. (env_manager.py:87)
  [2077 ms][env_manager] - INFO : Instantiating IGE object. (env_manager.py:88)
  [2077 ms][IsaacGymEnvManager] - INFO : Creating Isaac Gym Environment (IGE_env_manager.py:41)
  [2077 ms][IsaacGymEnvManager] - INFO : Acquiring gym object (IGE_env_manager.py:73)
  [2077 ms][IsaacGymEnvManager] - INFO : Acquired gym object (IGE_env_manager.py:75)
  [isaacgym:gymutil.py] Unknown args:  []
  [2078 ms][IsaacGymEnvManager] - INFO : Fixing devices (IGE_env_manager.py:89)
  [2078 ms][IsaacGymEnvManager] - INFO : Using GPU pipeline for simulation. (IGE_env_manager.py:102)
  [2078 ms][IsaacGymEnvManager] - INFO : Sim Device type: cuda, Sim Device ID: 0 (IGE_env_manager.py:105)
  [2078 ms][IsaacGymEnvManager] - INFO : Graphics Device ID: 0 (IGE_env_manager.py:119)
  [2078 ms][IsaacGymEnvManager] - INFO : Creating Isaac Gym Simulation Object (IGE_env_manager.py:120)
  [2078 ms][IsaacGymEnvManager] - WARNING : If you have set the CUDA_VISIBLE_DEVICES environment variable, please ensure that you set it
  to a particular one that works for your system to use the viewer or Isaac Gym cameras.
  If you want to run parallel simulations on multiple GPUs with camera sensors,
  please disable Isaac Gym and use warp (by setting use_warp=True), set the viewer to headless. (IGE_env_manager.py:127)
  [2078 ms][IsaacGymEnvManager] - WARNING : If you see a segfault in the next lines, it is because of the discrepancy between the CUDA device and the graphics device.
  Please ensure that the CUDA device and the graphics device are the same. (IGE_env_manager.py:132)
  Not connected to PVD
  +++ Using GPU PhysX
  Physics Engine: PhysX
  Physics Device: cuda:0
  GPU Pipeline: enabled
  [3195 ms][IsaacGymEnvManager] - INFO : Created Isaac Gym Simulation Object (IGE_env_manager.py:136)
  [3195 ms][IsaacGymEnvManager] - INFO : Created Isaac Gym Environment (IGE_env_manager.py:43)
  [3361 ms][env_manager] - INFO : IGE object instantiated. (env_manager.py:109)
  [3361 ms][env_manager] - INFO : Creating robot manager. (env_manager.py:118)
  [3361 ms][BaseRobot] - INFO : [DONE] Initializing controller (base_robot.py:26)
  [3361 ms][BaseRobot] - INFO : Initializing controller lee_attitude_control (base_robot.py:29)
  [3361 ms][base_multirotor] - WARNING : Creating 4096 multirotors. (base_multirotor.py:32)
  [3361 ms][env_manager] - INFO : [DONE] Creating robot manager. (env_manager.py:123)
  [3361 ms][env_manager] - INFO : [DONE] Creating simulation instance. (env_manager.py:125)
  [3361 ms][asset_loader] - INFO : Loading asset: quad.urdf for the first time. Next use of this asset will be via the asset buffer. (asset_loader.py:71)
  [3368 ms][env_manager] - INFO : Populating environment 0 (env_manager.py:179)
  [3496 ms][robot_manager] - WARNING : 
  Robot mass: 0.2500003944120692,
  Inertia: tensor([[8.4501e-04, 0.0000e+00, 3.3087e-24],
          [0.0000e+00, 8.4501e-04, 3.3087e-24],
          [3.3087e-24, 3.3087e-24, 1.6900e-03]], device='cuda:0'),
  Robot COM: tensor([[0.0000e+00, 0.0000e+00, 1.3411e-14, 1.0000e+00]], device='cuda:0') (robot_manager.py:427)
  [3496 ms][robot_manager] - WARNING : Calculated robot mass and inertia for this robot. This code assumes that your robot is the same across environments. (robot_manager.py:430)
  [3496 ms][robot_manager] - CRITICAL : If your robot differs across environments you need to perform this computation for each different robot here. (robot_manager.py:433)
  [3636 ms][env_manager] - INFO : Populating environment 1000 (env_manager.py:179)
  [3783 ms][env_manager] - INFO : Populating environment 2000 (env_manager.py:179)
  [3930 ms][env_manager] - INFO : Populating environment 3000 (env_manager.py:179)
  [4078 ms][env_manager] - INFO : Populating environment 4000 (env_manager.py:179)
  [4094 ms][env_manager] - INFO : [DONE] Populating environments. (env_manager.py:75)
  [4127 ms][IsaacGymEnvManager] - WARNING : Headless: False (IGE_env_manager.py:424)
  [4127 ms][IsaacGymEnvManager] - INFO : Creating viewer (IGE_env_manager.py:426)
  [4194 ms][IGE_viewer_control] - WARNING : Instructions for using the viewer with the keyboard:
  ESC: Quit
  V: Toggle Viewer Sync
  S: Sync Frame Time
  F: Toggle Camera Follow
  P: Toggle Camera Follow Type
  R: Reset All Environments
  UP: Switch Target Environment Up
  DOWN: Switch Target Environment Down
  SPACE: Pause Simulation
   (IGE_viewer_control.py:153)
  [4194 ms][IsaacGymEnvManager] - INFO : Created viewer (IGE_env_manager.py:432)
  *** Can't create empty tensor
  [4199 ms][asset_manager] - WARNING : Number of obstacles to be kept in the environment: 0 (asset_manager.py:32)
  WARNING: allocation matrix is not full rank. Rank: 4
  /home/tfly/temp/test_aeral_gym/aerial_gym_simulator/aerial_gym/control/motor_model.py:45: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
    torch.tensor(self.min_thrust, device=self.device, dtype=torch.float32).expand(
  /home/tfly/temp/test_aeral_gym/aerial_gym_simulator/aerial_gym/control/motor_model.py:48: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
    torch.tensor(self.max_thrust, device=self.device, dtype=torch.float32).expand(
  [4760 ms][control_allocation] - WARNING : Control allocation does not account for actuator limits. This leads to suboptimal allocation (control_allocation.py:48)
  [4761 ms][robot_manager] - ERROR : Not using warp. Initializing sensors (robot_manager.py:131)
</details>

**Comment by mihirk284** [comment]
Perhaps the rendering pipeline is set to the native Isaac Gym rendering pipeline?
This should not be an issue, however in my experience, for faster speed for training, I would recommend the Warp-based rendering pipeline.
